http-common/src/server.rs

```diff
@@ -182,7 +182,7 @@ macro_rules! make_service {
 
                                         http::Method::PUT => {
                                             let content_type = headers.get(hyper::header::CONTENT_TYPE).and_then(|value| value.to_str().ok());
-                                            if content_type.as_deref() != Some("application/json") {
+                                            if content_type.as_deref() != Some("application/json") && content_type.as_deref() != None {
```

Comment:
Better to make this be a `match` too instead of comparing twice. Same as the other methods, except the `_ =>` arm has `return Ok(Error { ... })` instead of `None`

Reply:
Fixed.